### PR TITLE
feat: getting started docs for Wyze

### PR DESCRIPTION
**Pull request by mc2147**

### CODE REVIEW

File: docs/device-guides/get-started-with-wyze-locks.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
+---
+description: Learn how to connect and control your Wyze locks with the Seam API.
+---
+
+# Get started with Wyze Oracode Locks
+
+[Image Here]
```
Comment (mc2147) [inline docs/device-guides/get-started-with-wyze-locks.md:7]: Need Getting Started Guide Cover here, e.g. Kwikset's:
image

File: docs/device-guides/get-started-with-wyze-locks.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+#### Authorize Your Workspace
+
+Navigate to the URL returned by the Webview object. Since you are using a sandbox workspace, complete the login flow by entering the Wyze [sandbox test account](https://docs.seam.co/latest/device-guides/sandbox-and-sample-data)credentials below:
+
+- **Wyze API Key ID:** ABC123ABC
+- **Wyze API Key** ABC123ABC
+- **email:** jane@example.com
+- **password:** 1234
+
+<figure><img src="[Wyze Connect Flow needed here]" alt=""><figcaption><p>Seam Connect Webview flow to connect Wyze account with Seam</p></figcaption></figure>
```
Comment (mc2147) [inline docs/device-guides/get-started-with-wyze-locks.md:132]: Need Wyze Connect Flow image here (e.g. Kwikset's):

File: docs/device-guides/get-started-with-wyze-locks.md [hunk 3]
```diff
@@ -0,0 +1,503 @@
+---
+description: Learn how to connect and control your Wyze locks with the Seam API.
+---
+
+# Get started with Wyze Oracode Locks
```
Comment (codetheweb) [inline docs/device-guides/get-started-with-wyze-locks.md:5]: 🙈 there's a few references to Oracode in here

File: docs/device-guides/get-started-with-wyze-locks.md [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+
+#### Obtain a Wyze API Key and Wyze API Key ID
+
+You will need create a Wyze API Key for your Wyze account. To see how to create a Wyze API Key for your Wyze account, [click here](https://support.wyze.com/hc/en-us/articles/16129834216731-Creating-an-API-Key).
+
+{% hint style="info" %}
+Note that both the API key and Key ID returned from Wyze are required to connect your Wyze account to Seam. Make sure to copy both values to somewhere secure!
+
+<img src="../.gitbook/assets/wyze-api-key-example.png" alt="Wyze API Key example" data-size="original">
+{% endhint %}
```
Comment (codetheweb) [inline docs/device-guides/get-started-with-wyze-locks.md:52]: this section is a little confusing in the context of a sandbox webview

File: docs/device-guides/get-started-with-wyze-locks.md (hunk 3 above)
Comment (mc2147) [reply docs/device-guides/get-started-with-wyze-locks.md]: Eek that was our latest integration so I used it as a template

File: docs/device-guides/get-started-with-wyze-locks.md (hunk 4 above)
Comment (mc2147) [reply docs/device-guides/get-started-with-wyze-locks.md]: Added a note for sandbox integration values, that should be more clear now

**Pull request merged**